audio/espeak-ng/Portfile

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                espeak-ng
+github.setup        espeak-ng espeak-ng 1.50
+version             1.50
+revision            1
+categories          audio
+platforms           darwin
+maintainers         nomaintainer
+license             GPL-3+
+
+description         eSpeak NG is a multi-lingual software speech synthesizer
+
+long_description    eSpeak NG is a multi-lingual software speech synthesizer
+
+homepage            https://github.com/espeak-ng/espeak-ng
+distname            ${version}
+checksums           rmd160  bd3dd4e17863bb1aa3a964517d888ed9997f0e3e \
+                    sha256  f0505366d5391f1291a9c5cc0816461719f46eaed84cfbf57fb81b0dc381de21 \
+                    size    13665736
+
+depends_lib         port:pcaudiolib \
+                    port:sonic \
+                    port:rb-ronn-ng \
+                    port:rb-kramdown \
+                    port:automake \
+                    port:autoconf \
+                    port:libtool \
+                    port:pkgconfig
+
+configure.cmd       ./autogen.sh && ./configure --with-extdict-ru --with-extdict-zh --with-extdict-zhy
+build.cmd           make src/espeak-ng src/speak-ng en && make
+build.target
+build.args-append   -i -k prefix=${prefix}
```

Comment:
Looks like `-i` ignores errors and `-k` keeps going. Why is this necessary? I would not think that we would want to ignore errors.

Is specifying prefix really necessary here? The build system should be remembering the prefix that was specified at configure time. If the build system is broken and is failing to do that, please make sure there's an upstream bug report about that and include the URL in a comment in the portfile.

Reply:
I found a way to avoid -i and -k args.

Reply:
I finally found out that this was related to espeak-ng not supporting parallel building. My solution on my system was to set '-j1', but is there another more preferable way to disable parallel building?

Reply:
`use_parallel_build no
`

[Here is the reference]([url](https://guide.macports.org/#reference.phases.build)) from the guide.